controlplane/nested/certificate/consts.go

```diff
@@ -36,8 +36,8 @@ const (
 	// KubeletClient defines the client cert name for kubelet.
 	KubeletClient secret.Purpose = "kubelet-client"
 
-	// ProxyClient defines the client cert name for the front proxy.
-	ProxyClient secret.Purpose = "proxy-client"
+	// FrontProxyClient defines the client cert name for the front proxy.
```

Comment:
Can we leave this as `proxy` this follows the base standards from CAPI see - https://cluster-api.sigs.k8s.io/tasks/certs/using-custom-certificates.html#using-custom-certificates

Reply:
Done